pkg/test/extensions/provider.go

```diff
@@ -80,9 +76,9 @@ func NewExternalBinaryProvider(logger *log.Logger, releaseImage,
 func (provider *ExternalBinaryProvider) Cleanup() {
 	if provider.tmpDir != "" {
 		if err := os.RemoveAll(provider.tmpDir); err != nil {
-			provider.logger.Printf("Failed to remove tmpDir %q: %v", provider.tmpDir, err)
+			logrus.Warningf("Failed to remove tmpDir %s: %v", provider.tmpDir, err)
```

Comment:
Is `warning` level good enough for this? Almost seems like this should be `error` level.